3-platform/order/domain/src/main/java/com/_6core/platform/orderdomain/model/Order.java

```diff
@@ -0,0 +1,15 @@
+package com._6core.platform.orderdomain.model;
+
+import java.math.BigInteger;
+import java.util.Set;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table(name = "orders")
+public class Order {
+  @Id private String id;
+  private BigInteger total;
+  private String status;
+  @Transient private Set<OrderItem> orderItems;
```

Comment:
Transient? why? 
```suggestion
  @Transient private Set<OrderItem> orderItems;
```

Reply:
this field looks like relation 1-n. 
@Transient marks field to be ignored by database\jpa repository.

Reply:
@sokimaaa because that field doesn't exist in 'orders' table

Reply:
okay